### ISSUE TITLE
Error while training my open cv model 

### ISSUE THREAD

**Issue by anishdevtech** [termux/termux-packages#17440]
In this i was trying to train a model on my mobile but it just get crash 
Aftec 5 to 6 try i get this error 
```
Error in cpuinfo: failed to parse the list of possible processors in /sys/devices/system/cpu/possible
Error in cpuinfo: failed to parse the list of present processors in /sys/devices/system/cpu/present
Error in cpuinfo: failed to parse both lists of possible and present processors
```
I have tried by making bash size less but still 

My code 

```py 
import os
import torch
import torch.nn as nn
import torch.optim as optim
from torch.utils.data import DataLoader
from torchvision import datasets, models, transforms

# Set the path to the training and validation directories
train_dir = './train'
val_dir = './val'

# Set the desired image size for your model
image_size = (224, 224)

# Define the transformation to apply to the images
transform = transforms.Compose([
    transforms.Resize(image_size),
    transforms.ToTensor(),
    transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
])

# Load the training and validation datasets
train_dataset = datasets.ImageFolder(train_dir, transform=transform)
val_dataset = datasets.ImageFolder(val_dir, transform=transform)

# Create data loaders for batching the data
batch_size = 32
train_loader = DataLoader(train_dataset, batch_size=batch_size, shuffle=True)
val_loader = DataLoader(val_dataset, batch_size=batch_size)

# Define the model architecture
model = models.resnet50(pretrained=True)
num_classes = len(train_dataset.classes)
model.fc = nn.Linear(model.fc.in_features, num_classes)

# Move the model to the GPU if available
device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
model = model.to(device)

# Define the loss function and optimizer
criterion = nn.CrossEntropyLoss()
optimizer = optim.SGD(model.parameters(), lr=0.001, momentum=0.9)

# Training loop
num_epochs = 10

for epoch in range(num_epochs):
    # Training phase
    model.train()
    train_loss = 0.0
    train_correct = 0

    for images, labels in train_loader:
        images = images.to(device)
        labels = labels.to(device)

        optimizer.zero_grad()
        outputs = model(images)
        _, predicted = torch.max(outputs.data, 1)
        loss = criterion(outputs, labels)

        loss.backward()
        optimizer.step()

        train_loss += loss.item() * images.size(0)
        train_correct += (predicted == labels).sum().item()

    train_loss /= len(train_dataset)
    train_accuracy = train_correct / len(train_dataset)

    # Validation phase
    model.eval()
    val_loss = 0.0
    val_correct = 0

    with torch.no_grad():
        for images, labels in val_loader:
            images = images.to(device)
            labels = labels.to(device)

            outputs = model(images)
            _, predicted = torch.max(outputs.data, 1)
            loss = criterion(outputs, labels)

            val_loss += loss.item() * images.size(0)
            val_correct += (predicted == labels).sum().item()

    val_loss /= len(val_dataset)
    val_accuracy = val_correct / len(val_dataset)

    # Print epoch statistics
    print(f"Epoch [{epoch+1}/{num_epochs}] - "
          f"Train Loss: {train_loss:.4f}, Train Accuracy: {train_accuracy:.4f} - "
          f"Val Loss: {val_loss:.4f}, Val Accuracy: {val_accuracy:.4f}")

# Save the trained model
torch.save(model.state_dict(), 'trained_model.pth')
```

**Comment by Grimler91** [comment]
Please provide the info about your environment and device that is requested in the [bug report template](https://github.com/termux/termux-packages/issues/new?assignees=&labels=untriaged%2Cbug+report&projects=&template=01-bug-report.yml&title=%5BBug%5D%3A+).

**Comment by anishdevtech** [comment]
Termux Variables:
TERMUX_API_VERSION=0.50.0
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=8861
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
arm
Subscribed repositories:
# sources.list
deb https://mirrors.bfsu.edu.cn/termux/apt/termux-main stable main
# root-repo (sources.list.d/root.list)
deb https://mirrors.bfsu.edu.cn/termux/apt/termux-root root stable
# sources.list.d/pointless.list
deb https://its-pointless.github.io/files/21 termux extras
Updatable packages:
clang/stable 16.0.6-1 arm [upgradable from: 16.0.6]
cmake/stable 3.26.4-1 arm [upgradable from: 3.26.4]
dbus/stable 1.15.6-1 arm [upgradable from: 1.15.6]
gettext/stable 0.21.1-1 arm [upgradable from: 0.21.1]
libarchive/stable 3.6.2-2 arm [upgradable from: 3.6.2-1]
libcompiler-rt/stable 16.0.6-1 arm [upgradable from: 16.0.6]
libllvm/stable 16.0.6-1 arm [upgradable from: 16.0.6]
libunbound/stable 1.17.1-2 arm [upgradable from: 1.17.1-1]
libxml2/stable 2.11.4-2 arm [upgradable from: 2.10.3-1]
lld/stable 16.0.6-1 arm [upgradable from: 16.0.6]
llvm/stable 16.0.6-1 arm [upgradable from: 16.0.6]
python-ensurepip-wheels/stable 3.11.4-2 all [upgradable from: 3.11.4-1]
python/stable 3.11.4-2 arm [upgradable from: 3.11.4-1]
unbound/stable 1.17.1-2 arm [upgradable from: 1.17.1-1]
termux-tools version:
1.38.3
Android version:
11
Kernel build information:
Linux localhost 4.4.177-21848546 termux/termux-packages#1 SMP PREEMPT Wed
Jul 13 14:26:29 KST 2022 armv8l Android
Device manufacturer:
samsung
Device model:
SM-A105F
LD Variables:
LD_LIBRARY_PATH=
LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec.so
Installed termux plugins:
com.termux.api versionCode:50

**Comment by anishdevtech** [comment]
```
Termux Variables:
TERMUX_API_VERSION=0.50.0
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=8861
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
arm
Subscribed repositories:
# sources.list
deb https://mirrors.bfsu.edu.cn/termux/apt/termux-main stable main
# root-repo (sources.list.d/root.list)
deb https://mirrors.bfsu.edu.cn/termux/apt/termux-root root stable
# sources.list.d/pointless.list
deb https://its-pointless.github.io/files/21 termux extras
Updatable packages:
clang/stable 16.0.6-1 arm [upgradable from: 16.0.6]
cmake/stable 3.26.4-1 arm [upgradable from: 3.26.4]
dbus/stable 1.15.6-1 arm [upgradable from: 1.15.6]
gettext/stable 0.21.1-1 arm [upgradable from: 0.21.1]
libarchive/stable 3.6.2-2 arm [upgradable from: 3.6.2-1]
libcompiler-rt/stable 16.0.6-1 arm [upgradable from: 16.0.6]
libllvm/stable 16.0.6-1 arm [upgradable from: 16.0.6]
libunbound/stable 1.17.1-2 arm [upgradable from: 1.17.1-1]
libxml2/stable 2.11.4-2 arm [upgradable from: 2.10.3-1]
lld/stable 16.0.6-1 arm [upgradable from: 16.0.6]
llvm/stable 16.0.6-1 arm [upgradable from: 16.0.6]
python-ensurepip-wheels/stable 3.11.4-2 all [upgradable from: 3.11.4-1]
python/stable 3.11.4-2 arm [upgradable from: 3.11.4-1]
unbound/stable 1.17.1-2 arm [upgradable from: 1.17.1-1]
termux-tools version:
1.38.3
Android version:
11
Kernel build information:
Linux localhost 4.4.177-21848546 termux/termux-packages#1 SMP PREEMPT Wed Jul 13 14:26:29 KST 2022 armv8l Android
Device manufacturer:
samsung
Device model:
SM-A105F
LD Variables:
LD_LIBRARY_PATH=
LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec.so
Installed termux plugins:
com.termux.api versionCode:50


```

**Comment by anishdevtech** [comment]
> Please provide the info about your environment and device that is requested in the [bug report template](https://github.com/termux/termux-packages/issues/new?assignees=&labels=untriaged%2Cbug+report&projects=&template=01-bug-report.yml&title=%5BBug%5D%3A+).

Termux Variables:
TERMUX_API_VERSION=0.50.0
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=8861
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
arm
Subscribed repositories:
# sources.list
deb https://mirrors.bfsu.edu.cn/termux/apt/termux-main stable main
# root-repo (sources.list.d/root.list)
deb https://mirrors.bfsu.edu.cn/termux/apt/termux-root root stable
# sources.list.d/pointless.list
deb https://its-pointless.github.io/files/21 termux extras
Updatable packages:
clang/stable 16.0.6-1 arm [upgradable from: 16.0.6]
cmake/stable 3.26.4-1 arm [upgradable from: 3.26.4]
dbus/stable 1.15.6-1 arm [upgradable from: 1.15.6]
gettext/stable 0.21.1-1 arm [upgradable from: 0.21.1]
libarchive/stable 3.6.2-2 arm [upgradable from: 3.6.2-1]
libcompiler-rt/stable 16.0.6-1 arm [upgradable from: 16.0.6]
libllvm/stable 16.0.6-1 arm [upgradable from: 16.0.6]
libunbound/stable 1.17.1-2 arm [upgradable from: 1.17.1-1]
libxml2/stable 2.11.4-2 arm [upgradable from: 2.10.3-1]
lld/stable 16.0.6-1 arm [upgradable from: 16.0.6]
llvm/stable 16.0.6-1 arm [upgradable from: 16.0.6]
python-ensurepip-wheels/stable 3.11.4-2 all [upgradable from: 3.11.4-1]
python/stable 3.11.4-2 arm [upgradable from: 3.11.4-1]
unbound/stable 1.17.1-2 arm [upgradable from: 1.17.1-1]
termux-tools version:
1.38.3
Android version:
11
Kernel build information:
Linux localhost 4.4.177-21848546 termux/termux-packages#1 SMP PREEMPT Wed Jul 13 14:26:29 KST 2022 armv8l Android
Device manufacturer:
samsung
Device model:
SM-A105F
LD Variables:
LD_LIBRARY_PATH=
LD_PRELOAD=/data/data/com.termux/files/usr/lib/libtermux-exec.so
Installed termux plugins:
com.termux.api versionCode:50
